src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -188,16 +191,19 @@ export const MetadataInstanceEditorCancelChanges: StoryObj<typeof MetadataSideba
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
 
-        const editButtons = await canvas.findAllByRole('button', { name: 'Edit' }, { timeout: 5000 });
+        // Edit buttons contains also template name
+        const editButton = await canvas.findByRole('button', { name: 'Edit My Template' }, { timeout: 5000 });
+        expect(editButton).toBeInTheDocument();
 
         let headlines = await canvas.findAllByRole('heading', { level: 1 });
         expect(headlines).toHaveLength(3);
         expect(headlines.map(heading => heading.textContent)).toEqual(
             expect.arrayContaining(['My Template', 'Select Dropdowns', 'Custom Metadata']),
         );
 
+        expect(editButton.ariaLabel).toBe('Edit My Template');
```

Comment:
I dont think we need to assert for it again, `findByRole` is using that label to locate that `editButton`